runtime/src/iree/modules/vmvx/module.c

```diff
@@ -675,42 +668,26 @@ IREE_VMVX_ABI_EXPORT(iree_vmvx_mmt4d_f32f32f32, mmt4d, v) {
                    /*stride1=*/1,
                    /*size0=*/M,
                    /*size1=*/N* out_tile_size);
-
-  unsigned accumulate_flag = args->flags & IREE_VMVX_MATMUL_FLAG_ACCUMULATE;
-  unsigned unhandled_flags = args->flags ^ accumulate_flag;
-  if (unhandled_flags) {
-    IREE_TRACE_ZONE_END(z0);
-    return iree_make_status(IREE_STATUS_INVALID_ARGUMENT,
-                            "unsupported mmt4d flags: 0x%x", unhandled_flags);
-  }
-
-  for (iree_host_size_t i = 0; i < M; ++i) {
-    for (iree_host_size_t j = 0; j < N; ++j) {
-      float* out_tile_ptr = out + i * out_stride0 + j * out_tile_size;
-      const float* lhs_panel_ptr = lhs + i * lhs_stride0;
-      const float* rhs_panel_ptr = rhs + j * rhs_stride0;
-      for (iree_host_size_t i0 = 0; i0 < M0; ++i0) {
-        for (iree_host_size_t j0 = 0; j0 < N0; ++j0) {
-          const float* lhs_tile_ptr = lhs_panel_ptr;
-          const float* rhs_tile_ptr = rhs_panel_ptr;
-          float* out_ptr = out_tile_ptr + i0 * N0 + j0;
-          float acc = accumulate_flag ? *out_ptr : 0.f;
-          for (iree_host_size_t k = 0; k < K; ++k) {
-            for (iree_host_size_t k0 = 0; k0 < K0; ++k0) {
-              float lhs_val = lhs_tile_ptr[i0 * K0 + k0];
-              float rhs_val = rhs_tile_ptr[j0 * K0 + k0];
-              acc += lhs_val * rhs_val;
-            }
-            lhs_tile_ptr += lhs_tile_size;
-            rhs_tile_ptr += rhs_tile_size;
-          }
-          *out_ptr = acc;
-        }
-      }
-    }
-  }
-
+  iree_ukernel_mmt4d_f32f32f32_params_t ukernel_params = {
+      .lhs_buffer = lhs,
+      .rhs_buffer = rhs,
+      .out_buffer = out,
+      .lhs_stride = lhs_stride0,
+      .rhs_stride = rhs_stride0,
+      .out_stride = out_stride0,
+      .M = M,
+      .N = N,
+      .K = K,
+      .M0 = M0,
+      .N0 = N0,
+      .K0 = K0,
+      .flags = args->flags};
```

Comment:
style: add a trailing , to keep the } on the next line (and make it easier to add/remove entries)

Reply:
done